app/assets/javascripts/Components/markdown_preview.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default class MarkdownPreview extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount = () => {
```

Comment:
There's something missing here compared to the old `updatePreview` method. Did you try testing the preview with annotations that use markdown syntax?